message_app_client/src/components/users_table/UsersTableDetails.js

```diff
@@ -0,0 +1,35 @@
+import React, { useState, useEffect } from "react";
+import TableRow from "./TableRow";
+
+const UsersTableDetail = (props) => {
+    const { userId } = props;
+    const [ users, setUsers ] = useState([]);
+    const [ isFetchSuccess, setIsFetchSuccess ] = useState(true);
+    const apiUrl = `http://localhost:5000/api/request/get_people/${userId}`;
```

Comment:
After we merge[ Long's PR](https://github.com/leth02/VL_Chat_App/pull/34), please register your API to `apiRoutes` in `state.js`